CHT/CHT.C

```diff
@@ -104,16 +104,14 @@ std::string preciceAdapter::CHT::ConjugateHeatTransfer::determineSolverType()
         DEBUG(adapterInfo("Did not find the transportProperties dictionary."));
     }
 
-    if (mesh_.foundObject<IOdictionary>(turbulenceModel::propertiesName))
+    if (mesh_.foundObject<IOdictionary>("turbulenceProperties"))
```

Comment:
The new name is `turbulenceModel::typeName`. However, hard-coding this makes it easier to work across OpenFOAM versions.